server/runtime/src/main/java/org/infinispan/server/logging/Messages.java

```diff
@@ -54,11 +54,11 @@
    @Message("Specifies a logging configuration file. Defaults to `%s`.")
    String serverHelpLoggingConfig(String defaultConfiguration);
 
-   @Message("Sets the name of the cluster. Defaults to `%s`.")
-   String serverHelpClusterName(String defaultClusterName);
+   @Message("Sets the name of the cluster. Default set by configuration expression")
+   String serverHelpClusterName();
 
-   @Message("Specifies the JGroups stack for clustering. Defaults to `%s`.")
-   String serverHelpClusterStack(String defaultStack);
+   @Message("Specifies the JGroups stack for clustering. Default set by configuration expression")
+   String serverHelpClusterStack();
```

Comment:
I disagree with this change. If you remove the expression, it will default to `tcp`.